sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2613,6 +2625,62 @@ def expand(self, input):
       # communicate to Java that this write should use dynamic destinations
       table = StorageWriteToBigQuery.DYNAMIC_DESTINATIONS
 
+    use_cdc_writes = False
+    # if CDC functionality is configured we need to check if a callable has
+    # been passed to extract MutationInfo from the rows to be written
+    if callable(self._use_cdc_writes):
```

Comment:
I'm realizing that this callable CDC writes option is more complex than I thought. For it to be complete and not confusing for users, we will have to provide this logic for both Python dict inputs and Beam Row inputs

I suggest we keep the CDC option to a boolean and make this improvement in a future PR

Reply:
Let me try to have it implemented for both in this change, if you still see it incomplete I will remove it after the next review.

Reply:
removed `Callable` argument for now, will work on it on a separated PR.